core/src/main/java/org/apache/gravitino/connector/WildcardPropertiesMetadata.java

```diff
@@ -0,0 +1,175 @@
+
+/**
+ * WildcardPropertiesMeta is a metadata interface for defining wildcard properties in the properties
+ * metadata. <br>
+ * <br>
+ * WildcardPropertiesMetadata interface defines: <br>
+ * Prefix.Wildcard = "WildcardValue1,WildcardValue2" <br>
+ * Prefix.*.properties-key1 = "" <br>
+ * Prefix.*.properties-key2 = "" <br>
+ * NOTE: Prefix name support multiple segment, separated by dot. for example: "a1.b1.c1" <br>
+ * NOTE: Suffix properties-key{N} support multiple segment, separated by dot. for example:
+ * "x1.y1.z1" <br>
+ * <br>
+ * Use define a WildcardPropertiesMetadata object: <br>
+ * a1.b1.c1.Wildcard = "WildcardValue1,WildcardValue2" <br>
+ * a1.b1.c1.{WildcardValue1}.x1.y1.z1 = "WildcardValue1 property value" <br>
+ * a1.b1.c1.{WildcardValue1}.x1.y2.z2 = "WildcardValue1 property value" <br>
+ * a1.b1.c1.{WildcardValue1}.x1.y2.z3 = "WildcardValue1 property value" <br>
+ * a1.b1.c1.{WildcardValue2}.x1.y1.z1 = "WildcardValue2 property value" <br>
+ * a1.b1.c1.{WildcardValue2}.x1.y2.z2 = "WildcardValue2 property value" <br>
+ * a1.b1.c1.{WildcardValue2}.x1.y2.z3 = "WildcardValue2 property value" <br>
+ * <br>
+ * Configuration Example: {@link AuthorizationPropertiesMetadata}, <br>
+ * The Prefix is "authorization.chain", <br>
+ * The Wildcard is "plugins" <br>
+ * "authorization.chain.plugins" = "hive1,hdfs1" <br>
+ * "authorization.chain.hive1.provider" = "ranger"; <br>
+ * "authorization.chain.hive1.catalog-provider" = "hive"; <br>
+ * "authorization.chain.hive1.ranger.auth.type" = "simple"; <br>
+ * "authorization.chain.hive1.ranger.admin.url" = "http://localhost:6080"; <br>
+ * "authorization.chain.hive1.ranger.username" = "admin"; <br>
+ * "authorization.chain.hive1.ranger.password" = "admin"; <br>
+ * "authorization.chain.hive1.ranger.service.name" = "hiveDev"; <br>
+ * "authorization.chain.hdfs1.provider" = "ranger"; <br>
+ * "authorization.chain.hdfs1.catalog-provider" = "hadoop"; <br>
+ * "authorization.chain.hdfs1.ranger.auth.type" = "simple"; <br>
+ * "authorization.chain.hdfs1.ranger.admin.url" = "http://localhost:6080"; <br>
+ * "authorization.chain.hdfs1.ranger.username" = "admin"; <br>
+ * "authorization.chain.hdfs1.ranger.password" = "admin"; <br>
+ * "authorization.chain.hdfs1.ranger.service.name" = "hdfsDev"; <br>
+ */
+public interface WildcardPropertiesMetadata {
+  String WILDCARD = "*";
+  String WILDCARD_CONFIG_VALUES_SPLITTER = ",";
+
+  /** The prefix name */
+  String prefixName();
+
+  /** The WildcardNode define name */
+  String wildcardName();
+
+  /** The `Prefix.Wildcard` properties key name */
+  default String wildcardPropertyKey() {
+    return String.format("%s.%s", prefixName(), wildcardName());
+  }
+
+  /** Get the property value by wildcard value and property key */
+  default String getPropertyValue(String wildcardValue, String propertyKey) {
+    return String.format("%s.%s.%s", prefixName(), wildcardValue, propertyKey);
+  }
```

Comment:
Are there more suitable names for these functions? It's confused now

Reply:
OK, I improve this function name.